TripKit/Classes/helpers/TKAccessibilityHelper.swift

```diff
@@ -0,0 +1,48 @@
+//
+//  TKAccessibilityHelper.swift
+//  TripKit-iOS
+//
+//  Created by Kuan Lun Huang on 22/2/18.
+//  Copyright © 2018 SkedGo. All rights reserved.
+//
+
+import Foundation
+import RxSwift
+
+@objc public class TKAccessibilityHelper: NSObject {
+  
+  let region: SVKRegion
+  
+  @objc public var isWheelchairAvailable: Bool {
+    return rx_wheelchairAvailable.value
+  }
+  
+  private let rx_regionInfo: Variable<API.RegionInfo?> = Variable(nil)
+  private let rx_wheelchairAvailable = Variable(false)
+  public let rx_wheelchairAvailabilityUpdated = PublishSubject<Bool>()
+  
+  private let disposeBag = DisposeBag()
+  
+  // MARK: -
+  
+  @objc public init(region: SVKRegion) {
+    self.region = region
+    
+    super.init()
+    
+    rx_regionInfo.asObservable()
```

Comment:
Why does that need to be a variable? Why not just call fetch and share the last value?

Reply:
Stylistically, it’s weird to make a network request on allocating an object. It’s better to just do set up in `init` and have a method to fetch on demand.

Reply:
Simplified